### DISCUSSION TITLE
adapter for lit

### DISCUSSION THREAD

**Discussion by chenshuichao** [DouyinFE/semi-design#243] [Ideas]
foundation and adapter设计的非常巧妙。
web component是大趋势，建议尽快推出adapter for lit。
lit是一个基于web component的framework。

**Comment by pointhalo** [comment]
https://github.com/DouyinFE/semi-design/blob/02401f10d58014a1c6f20d628912f007c67b7eb0/content/start/introduction/index.md?plain=1#L109

【我们团队当前阶段重点会聚焦于 React 体系内，但 WebComponent 也是我们重点关注的方向之一。未来时机合适，我们会进行更多的尝试，敬请期待。】

感谢你的建议，Lit 这个技术栈本身也是我们重点关注的方向之一。但由于人力因素以及我们团队在我们公司内部需要对现有业务提供更多支持，我们目前的重点工作依然会围绕React体系展开。  

未来我们其他方向的工作逐渐稳定后，我们会有更多的尝试，短期内（一到两个双月内）暂时不会有这方面的计划。

**Comment by xmsz** [comment]
今年有计划吗？我最近用了shoelace，感觉很不错，本来以为会有很多坑，结果意外的成熟

1. 接入react，没有问题，就引用包
2. 主题也有，覆盖样式也能做到
3. 事件监听也没问题
4. 调试也没问题
5. 兼容，一般b端后不用太在意

我觉得确实非常好，这样团队不管用什么框架都能用到同一套UI组件，并且UI组件只需要写一次，对于业务组件真的太爽了

![localhost_3001__id=15](https://github.com/DouyinFE/semi-design/assets/5879146/8cc5531a-7e95-47ce-9c76-9d0a75319102)

**Reply by pointhalo** [reply]
目前没有这种计划。我们团队当前的重点在设计稿转代码 DesigntoCode （https://semi.design/code） 事项上
